src/pages/Product/Product.js

```diff
@@ -1,30 +1,37 @@
-import React from "react";
+import React, { useState } from "react";
+import Color from "./components/Color/Color";
+import ColorButton from "./components/ColorButton/ColorButton";
+import Count from "./components/Count/Count";
+import Review from "./components/Review/Review";
 import "./Product.scss";
 
 const Product = () => {
+  const [color, setColor] = useState("white");
+  const [review, setReview] = useState("더보기 ▼ ");
```

Comment:
```suggestion
  const [review, setReview] = useState(false);
```
리뷰에 담긴 값은 string이 아닌 boolean 값이 되어야 합니다!